reference/build_helpers/visual_studio.rst

```diff
@@ -84,20 +84,22 @@ Builds Visual Studio project with the given parameters.
 
 Parameters:
     - **project_file** (Required): Path to the *.sln* file.
-    - **targets** (Optional, Defaulted to ``None``): List of targets to build.
-    - **upgrade_project** (Optional, Defaulted to ``True``): Will call :command:`devenv` to upgrade the solution to your current Visual Studio.
-    - **build_type** (Optional, Defaulted to ``None``): Use a custom build type name instead of the default ``settings.build_type`` one.
-    - **arch** (Optional, Defaulted to ``None``): Use a custom architecture name instead of the ``settings.arch`` one.
-      It will be used to build the ``/p:Configuration=`` parameter of :command:`MSBuild`.
-      It can be used as the key of the **platforms** parameter. E.g. ``arch="x86", platforms={"x86": "i386"}``
-    - **force_vcvars** (Optional, Defaulted to ``False``): Will ignore if the environment is already set for a different Visual Studio
-      version.
+    - **targets** (Optional, Defaulted to ``None``): Sets ``/target`` flag to the specified list of targets to build.
+    - **upgrade_project** (Optional, Defaulted to ``True``): Will call :command:`devenv /upgrade` to upgrade the solution to your current
+      Visual Studio.
+    - **build_type** (Optional, Defaulted to ``None``): Sets ``/p:Configuration`` flag to the specified value. Use a custom build type name
+      instead of the default ``settings.build_type`` one.
```

Comment:
**build_type** (Optional, Defaulted to ``None``): Sets ``/p:Configuration`` flag to the specified value, it will overrride the value from `settings.build_type`.